Add graphql to api token documentation (#1287)
strapi/documentation: docs/developer-docs/latest/plugins/graphql.md

:::

Then on each request, send along an `Authorization` header in the form of `{ "Authorization": "Bearer YOUR_JWT_GOES_HERE" }`. This can be set in the HTTP Headers section of your GraphQL Playground.

> ### API tokens

Reviewer: Should this be at the `##` level so that it doesn't fall under the "Users and Permissions Plugin" section, since API tokens are an alternate option to the plugin?

Although, I wonder if this could be combined or simplified somehow (maybe outside the scope of what you're doing, but a future note).  That is, the authorization header works exactly the same whether it's a JWT or API Token. So we could have a section about "Adding an authorization header" that explains the bearer token is either an API Token, or the JWT returned from a login.

Author: You are correct, I fixed the header level. 

On the second point, I agree and it is something we can revisit in the next quarter. We have been discussing a concepts section with @Aurelsicoko and one of the topics that I think we should cover early on is authentication/authorization.

> To use API tokens for authentication pass the token in the `Authorization` header using the format `bearer your-api-token`.

Reviewer: ```suggestion
To use API tokens for authentication, pass the token in the `Authorization` header using the format `Bearer your-api-token`.
```

:::note

> Using API tokens in the the GraphQL playground requires a slightly different syntax. In the `HTTP HEADERS` tab add the following code:

Reviewer: ```suggestion
Using API tokens in the the GraphQL playground requires adding the authorization header with your token in the `HTTP HEADERS` tab:
```

The syntax should be the same as if the graphql queries are being made programmatically, so maybe something like this?

```http
{

> "authorization" : "Bearer <TOKEN>"

Reviewer: ```suggestion
  "Authorization" : "Bearer <TOKEN>"
```
